docs/oss_rules_of_play_rating.md

```diff
@@ -0,0 +1,65 @@
+# Rules of play ratings for open source projects
+
+This section describes a rules of play rating for open source projects.
+The rating is implemented in the [OssRulesOfPlayRating](https://github.com/SAP/fosstars-rating-core/blob/master/src/main/java/com/sap/oss/phosphor/fosstars/model/rating/oss/OssRulesOfPlayRating.java) class.
+
+The rating assesses how well projects care about open source best practices and community health aspects. 
+The rating may be used to estimate how well-administered an open source project is and how it adheres to recommendations how an open source project should be set up.
+
+[By definition](ratings.md), the rules of play rating produces a score, a label and a confidence score - the same way as the security score.
+Here is a list of labels:
+
+1.  `PASSED`: the project implements good measures and in general cares about open source rules of play.
+1.  `PASSED_WITH_WARNING`: the compliance to open source rules of play in the project fulfills the minimum expectations, but shows some issues that should be resolved.
+1.  `FAILED`: looks like the project doesn't care well about open source rules of play.
+1.  `UNCLEAR`: there is no enough data to reliably calculate a score and a label for the project.
```

Comment:
Before we provide this information on what the rules of play rating should be viewed as, can we provide the list of Rules?

Reply:
`UNCLEAR`: Not enough information to reliably calculate a score and a label for the project.